tests/Unit/Elliptic/Triggers/Test_EveryNIterations.cpp

```diff
@@ -19,7 +23,6 @@
 namespace {
 struct IterationIdTag : db::SimpleTag {
   using type = size_t;
-  static std::string name() noexcept { return "IterationIdTag"; }
```

Comment:
And now you don't need `<string>` anymore.

Reply:
is it not needed for the call to `test_factory_creation`?

Reply:
You're right, it is.  IWYU is confused, I guess.